apps/condo/domains/meter/schema/Meter.js

```diff
@@ -165,6 +166,22 @@ const Meter = new GQLListSchema('Meter', {
             isRequired: true,
             defaultValue: false,
         },
+        b2cApp: {
```

Comment:
We need to restrict access to update/create this field

Reply:
miniapps use OIDC authorization to perform actions "on behalf of the user". Therefore, I still have no idea how you can cut the rights in this place.